toolchains/gcc_compatibles.cmake

```diff
@@ -22,9 +23,16 @@ set( TNUN_compiler_exceptions_off      -fno-exceptions                       )
 set( TNUN_compiler_optimize_for_speed  -O3                                   )
 set( TNUN_compiler_optimize_for_size   -Os                                   )
 set( TNUN_compiler_report_optimization -ftree-vectorizer-verbose=6           )
-set( TNUN_compiler_release_flags       -DNDEBUG -fomit-frame-pointer -ffunction-sections -fdata-sections -fmerge-all-constants -fno-stack-protector )
+set( TNUN_compiler_release_flags       -DNDEBUG -fomit-frame-pointer -ffunction-sections -fmerge-all-constants -fno-stack-protector )
+if( ${CMAKE_SYSTEM_NAME} MATCHES "Linux" )
+    set( TNUN_linker_release_flags         -Wl,--gc-sections                     )
+endif()
+set( TNUN_default_warnings             -Wall -Wextra -Wstrict-aliasing       )
+set( TNUN_warnings_as_errors           -Werror                               )
+set( TNUN_disabled_warnings            -Wno-unknown-pragmas -Wno-unused-parameter -Wno-ignored-qualifiers -Wno-sign-conversion -Wno-clobbered -Wno-format-zero-length -Wno-error=deprecated-declarations     )
```

Comment:
where do you have problems with Wclobbered?
Wunused-parameter is not a trivial warning and i don't want it disabled by default...but again why the introduction of this TNUN_disabled_warnings variable?